### ISSUE TITLE
doesnt even start also happens with 0.0.4

### ISSUE THREAD

**Issue by UttamOnTop** [Luna5ama/TrollHack#32]
[crash-2022-08-12_19.06.25-client.txt](https://github.com/Luna5ama/TrollHack/files/9330588/crash-2022-08-12_19.06.25-client.txt)

**Comment by Luna5ama** [comment]
`C:\Users\ammoc\AppData\Roaming\Minecraft%20Instances\Troll\mods\TrollHack-0.0.3.jar (The system cannot find the path specified)`
Removes the space in your path or use 0.0.4

**Comment by UttamOnTop** [comment]
[crash-2022-08-14_08.58.55-client.txt](https://github.com/Luna5ama/TrollHack/files/9333672/crash-2022-08-14_08.58.55-client.txt)
still doesnt work also using 0.0.4

**Comment by Luna5ama** [comment]
Send `latest.log` from `.minecraft/log`

**Comment by UttamOnTop** [comment]
[latest.log](https://github.com/Luna5ama/TrollHack/files/9334328/latest.log)

**Comment by Luna5ama** [comment]
Will be fixed in 0.0.5

For now just go to `.minecraft/trollhack/config/modules/default.json`, find `FastRender`, you should be seeing something like this:
```
{
  "name": "FastRender",
  "settings": {
    "enabled": true,
    "bind": "Minus",
    "on_hold": "OFF",
    "visible": true,
    "default": false
  }
}
```
Change `true` in `"enabled": true` to false, like `"enabled": false`

**Comment by TwoThreeTwo** [comment]
Luna best support
